### DISCUSSION TITLE
Suggestion: Add an option to collapse all namespace

### DISCUSSION THREAD

**Discussion by ANNOARK** [YU000jp/logseq-plugin-short-namespaces#8] [General]
`Example/Look/Like -> .../Like`
Looks like another method [Collapse namespace](https://github.com/cannibalox/logseq-custom-files/tree/main#namespace-prefixes-collapser-)

**Comment by YU000jp** [comment]
@ANNOARK 
Actually, if you choose `Abbreviate(..)` and `1 level` in the plugin settings, it will display something similar to `../../../Like`.
Are you saying that you want to change it to `../Like`?

**Reply by mcules** [reply]
Would be nice to have it like ../Like instead of ../../../Like.

**Reply by rowsof** [reply]
Love this plugin, thank you for the work. I'd love it if it could be taken a step further even, to completely hide everything except the intended page's name. 

So for example, the user should be able to select between:

```../../../Like```
```../Like```
```Like``` <-- the one I want 🥳

**Reply by YU000jp** [reply]
> Love this plugin, thank you for the work. I'd love it if it could be taken a step further even, to completely hide everything except the intended page's name. 
> 
> So for example, the user should be able to select between:
> 
> ```../../../Like```
> ```../Like```
> ```Like``` <-- the one I want 🥳

OK. I will release a next version.

**Reply by YU000jp** [reply]
DONE
https://github.com/YU000jp/logseq-plugin-short-namespaces/issues/24
